package.json

```diff
@@ -12,10 +12,12 @@
         "classnames": "^2.2.6",
         "lodash": "^4.17.15",
         "node-sass": "^4.13.1",
+        "pdfjs-dist": "https://github.com/veraPDF/pdfjs-dist/tarball/3b11077ff115c7f88e4b0accd79511d1b62ab722",
```

Comment:
Please use `npm install github:<githubname>/<githubrepo>[#<commit-ish>]` format for this dependency, 
where `#<commit-ish>` is out tag name